### PR TITLE
Update set-up-multi-factor-authentication.md

### PR DESCRIPTION
**Pull request by msbemba**
Added steps to Turn on Per person MFA

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9  Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2098

File: microsoft-365/admin/security-and-compliance/set-up-multi-factor-authentication.md [hunk 1]
```diff
@@ -91,6 +91,14 @@ If you have previously turned on per person MFA, you must turn it off before ena
 1. On the **Active users** page, choose **Multi-factor authentication**.
 1. On the multi-factor authentication page, select each user and set their Multi-Factor auth status to **Disabled**.
 
+### Turn On Per person MFA
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/security-and-compliance/set-up-multi-factor-authentication.md]:
```suggestion
### Turn on per-user MFA
```

File: microsoft-365/admin/security-and-compliance/set-up-multi-factor-authentication.md [hunk 2]
```diff
@@ -91,6 +91,14 @@ If you have previously turned on per person MFA, you must turn it off before ena
...
+
+If you have security defaults turned Off and donot want to turn on MFA for the entire organization
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/security-and-compliance/set-up-multi-factor-authentication.md]:
```suggestion
If you have security defaults turned off and do not want to turn on MFA for the entire organization:
```

File: microsoft-365/admin/security-and-compliance/set-up-multi-factor-authentication.md [hunk 3]
```diff
@@ -91,6 +91,14 @@ If you have previously turned on per person MFA, you must turn it off before ena
...
+
+1. In the Microsoft 365 admin center, in the left nav choose **Users** > **Active users**.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/security-and-compliance/set-up-multi-factor-authentication.md]:
```suggestion
1. In the Microsoft 365 admin center, in the left nav, choose **Users** > **Active users**. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Sirkkuw Please merge this PR. Thanks.

**Comment by Sirkkuw** [comment]
@yogkumgit - we don't want this info there. this is going to go away and PG doesn't want anyone using the per person MFA.

**Comment by msbemba** [comment]
Thank you @Sirkkuw

**Pull request closed without merging**